pom.xml

```diff
@@ -69,6 +69,7 @@
         <shiro.previousVersion>1.7.1</shiro.previousVersion>
         <!-- Replaced by the build number plugin at build time: -->
         <buildNumber>${user.name}-${maven.build.timestamp}</buildNumber>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
```

Comment:
I think the recommendation is to the the time of the commit when getting started.

Reply:
@bmarwell you mean setting this property with the original commit date of the creation of the pom.xml?

Reply:
No I mean like: 2021-07-19. But "1" or "0" will also do.

Reply:
Ok, I updated the timestamp with the commit timestamp.